### DISCUSSION TITLE
note of frustration

### DISCUSSION THREAD

**Discussion by halexandrino** [flet-dev/flet#3697] [General]
Oh my god! I only need to access the csv files from one directory. Why does this seem absurd on Android? 
I developed an app for salt analysis for a client, using random forest as ML model. I made the interface using tkinter (and it works great). My problem... Now the client wants an Android version for the same app. Since then I've tried everything... kivy, toga (and beeware), chaquopy, flet and kotlin. and in two years, I was unable to access the files in the folder. I learned everything else, except the simple one, how to get access to files in a folder (storage/0/downloads e.g.).
There are only futile things... to do list, image viewer... but a simple csv file read app examples... nothing.

**Comment by FeodorFitsner** [comment]
Sorry about having a bad experience with Flet! We have an open issue for your problem: https://github.com/flet-dev/flet/issues/2357 which has a high priority. You may add more specific details/request to that issue.
